### PR TITLE
Preparation for BDD tests unification

### PR DESCRIPTION
**Pull request by Tatarinho**
There is a need for small adjustments in bdd host-steps and utils to provide working tests in next PRs.

### CODE REVIEW

File: bdd/lib/utils.ts [hunk 1]
```diff
@@ -194,10 +194,13 @@ export async function waitUntilStreamEquals(stream: Readable, expected: string,
                 response += chunk;
 
                 if (response === expected) return expected;
+                if (response.length >= expected.length) {
+                    assert.equal(response, expected);
+                }
```
Comment (jan-warchol) [inline bdd/lib/utils.ts:199]: maybe I'm missing something, but isn't this redundant? we have previous check for the situation when lengths are equal, and when they are unequal, the assertion will fail.

**Review by jan-warchol — COMMENTED** [review]
I don't see anything wrong, but I'm not sure what this change gives us. Could you explain?

**Comment by Tatarinho** [comment]
> I don't see anything wrong, but I'm not sure what this change gives us. Could you explain?

Change in host-steps.ts is related to js sequences with debug args from this pr: https://github.com/scramjetorg/reference-apps/pull/2

Without it we are not able to read output response. As we want to have unified tests for python and js sequences, we need some extra adjustments also in hoststeps etc. to be sure everything works the same for both languages.

File: bdd/lib/utils.ts (hunk 1 above)
Comment (Tatarinho) [reply bdd/lib/utils.ts:199]: For now we don't see reason why it failed.

**Review by alicja-gruzdz — APPROVED** [review]
As extra we could check if each chunk sub-matches `expected` and break if not

**Pull request merged**